src/libs/isHexColor.ts

```diff
@@ -0,0 +1,10 @@
+// @ts-ignore allowing typedoc to build
+import assertString from '../utils/assertString.ts';
+
+// supports 6 digit and 3 digit hex colors
+const hex = /^#([0-9a-f]{6}|[0-9a-f]{3})$/i;
+
+export const isBtcAddress = (str: string) => {
```

Comment:
@hamptonmoore function name `isBtcAddress` is incorrect. It should be `isHexColor`?

Reply:
I don't even know how I made that mistake, sorry. I'll fix